### PR TITLE
Return arch version to admin

### PR DESCRIPTION
**Pull request by mjudeikis**
Returns arch version via Admin action to help with scripting and insights.

### CODE REVIEW

File: pkg/api/admin/openshiftcluster.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ type OpenShiftCluster struct {
 
 // OpenShiftClusterProperties represents an OpenShift cluster's properties.
 type OpenShiftClusterProperties struct {
-	ArchitectureVersion     ArchitectureVersion     `json:"architectureVersion,omitempty"`
+	ArchitectureVersion     ArchitectureVersion     `json:"architectureVersion"` // ArchitectureVersion is int so 0 is valid value to be returned
```
Comment (ihcsim) [inline pkg/api/admin/openshiftcluster.go:31]: >  // ArchitectureVersion is int so 0 is valid value to be returned

To confirm my understanding, `0` is `ArchitectureVersion1` and `1` is `ArchitectureVersion2`?

Comment (mjudeikis) [reply pkg/api/admin/openshiftcluster.go:31]: yes :) we needed default for all clusters without need to go and fix all of them :)

Comment (Makdaam) [reply pkg/api/admin/openshiftcluster.go:31]: Can we skip ArchitectureVersion1 and go straight from 0 to 2 or realign the numbers and names somehow? It's easier to make a mistake with this kind of misalignment.

Comment (mjudeikis) [reply pkg/api/admin/openshiftcluster.go:31]: Dont think so :/

Comment (ihcsim) [reply pkg/api/admin/openshiftcluster.go:31]: If we really want to realign the int value with the version, we can try setting `ArchitectureVersionV1 ArchitectureVersion = iota + 1 [here](https://github.com/Azure/ARO-RP/blob/00f28f958ab5e2024f2478ef5445087023a0db5f/pkg/api/admin/openshiftcluster.go#L181). I want to say it looks like a safe change, but not :100: sure.

**Review by ihcsim — APPROVED** [review]
LGTM.

**Review by olga-mir — APPROVED** [review]

**Pull request merged**